cowprotocol/cowswap
# 411/widget functionality

# Summary

Part of #411 

Actually, not doing anything that I mentioned there, this is just a tiny modding change to change the text, icon and link in the footer to Cow protocol instead of uniswap.

  # To Test

1. Run `yarn widget:start`
2. Open `localhost:5001`
3. Head to `lib/components/swap/swap`
![Screen Shot 2022-04-18 at 17 06 50](https://user-images.githubusercontent.com/43217/163837069-bea7f142-0eaf-4b35-a774-6eab558db5db.png)
* In the loaded swap component, at the bottom, you should see `Powered by CoW Protocol`
![Screen Shot 2022-04-18 at 17 06 56](https://user-images.githubusercontent.com/43217/163837144-6dc6cfbe-68dc-4f34-8148-f41757120321.png)
